setup.py

```diff
@@ -1,13 +1,14 @@
 from setuptools import setup
 
-pennylane_devices_list=[
+pennylane_devices_list = [
     'synqs.sqs = pennylane_ls:SingleQuditDevice',
-    'synqs.mqs = pennylane_ls:MultiQuditDevice'
+    'synqs.mqs = pennylane_ls:MultiQuditDevice',
+    'synqs.ls = pennylane_ls:FermionDevice',
```

Comment:
should be synqs fs for fermion simulator no ?

Reply:
Agree. synqs.fs would be nice.